L1Trigger/L1TGlobal/src/CorrCondition.cc

```diff
@@ -390,6 +400,9 @@ const bool l1t::CorrCondition::evaluateCondition(const int bxEval) const {
         eta0Phy = 0.5 * (binEdges.second + binEdges.first);
         binEdges = m_gtScales->getMUScales().etBins.at(etBin0);
         et0Phy = 0.5 * (binEdges.second + binEdges.first);
+	// Added by R.Cavanaugh for displaced muons
```

Comment:
Please remove meaningless comments. Git versioning system keeps track of authors of commits.

Reply:
@rekovic 
Many thanks for the helpful request to remove comments.  Done.